CMakeLists.txt

```diff
@@ -7,9 +7,14 @@ CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
 
 PROJECT(OpenTomb)
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -msse3")
+IF( ${CMAKE_COMPILER_IS_GNUCC} )
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -msse3")
+ENDIF()
+IF( ${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
```

Comment:
you can combine both IFs with an `OR`. And enclose the `${...}` with double-quotes.